BlockSettleUILib/AuthAddressConfirmDialog.cpp

```diff
@@ -102,12 +102,36 @@ void AuthAddressConfirmDialog::onError(const QString &errorText)
    reject();
 }
 
-void AuthAddressConfirmDialog::onAuthAddressSubmitError(const QString &address, const QString &error)
+void AuthAddressConfirmDialog::onAuthAddressSubmitError(const QString &address, const bs::error::AuthAddressSubmitResult statusCode)
 {
    progressTimer_.stop();
+
+   QString errorText;
+
+   switch (statusCode) {
+   case bs::error::AuthAddressSubmitResult::SubmitLimitExceeded:
+      errorText = tr("Your account has reach the limit of how many Authentication Addresses it may have in submitted state. Please verify your currently submitted addresses before submitting further addresses");
+      break;
+   case bs::error::AuthAddressSubmitResult::ServerError:
+      errorText = tr("Server error. Please try again later.");
```

Comment:
Some texts are with periods, some are not. I think we can have all of them without periods for better readability. E.g. this one can be transformed to "Server error - please try again later".

Reply:
Dots to end added where missing. Don't think that dash could improve readability of 6 words.